### ISSUE TITLE
[QUESTION] Nested Grid problems (potential numerical instabilities)

### ISSUE THREAD

**Issue by lfreese** [geoschem/geos-chem#605]
Hi all-- 

I've been working on nested grid simulations in NA and am consistently getting what seem like large numerical instabilities in high latitudes. I've attached the images of O3 and SO2 for examples. 

This run is 6 months of spinup (which all looked reasonable across 2015), but begins to have large values (eg: NOx values of >1e20) in one gridbox in the north-eastern part of the run. It then continues into 2016, where the error blows up sometime in March (see attached image). 

I had reported issues with the MERRA2 data and small negative numbers in #501 and had fixed that as explained there. However, it seems that was pointing towards a much large issue of numerical instability. 

I'm using v12.9.3 with only the above modifications made, and GEOS_0.5x0.625/MERRA2 files. This happens across 5 different runs all of which I have slightly different HEMCO emissions inputs for the NEI files. 

Attached here are (for one example run)
1. The HEMCO_Config.rc file
2. The input.geos file

The model does continue to run, so there aren't any useful error files, aside from the ones posted in #501 

It does look like there is something going on with the boundary files, as the edges have quite a lot of numerical noise. I was suggested to reach out to @msulprizio since you've done so much of the flexgrid work, and maybe it is something to do with the nested grid?

Thanks!
![Screen Shot 2021-02-03 at 11 13 58 AM](https://user-images.githubusercontent.com/50365319/106775671-52872d80-6611-11eb-8807-9163f67d8d80.png)
![Screen Shot 2021-02-03 at 11 13 30 AM](https://user-images.githubusercontent.com/50365319/106775686-55821e00-6611-11eb-8c77-3db53f44c452.png)
[input.geos.txt](https://github.com/geoschem/geos-chem/files/5919898/input.geos.txt)
[HEMCO_Config.txt](https://github.com/geoschem/geos-chem/files/5919900/HEMCO_Config.txt)

**Comment by msulprizio** [comment]
Are these high levels occurring at all levels, a particular level, or in the column mean? See [this post on the wiki](http://wiki.seas.harvard.edu/geos-chem/index.php/MERRA-2_implementation_details#Default_timestep_for_MERRA-2_nested_grid_simulations) describing a similar issue that Bram Maasakkers and I ran into when running nested NA simulations with MERRA-2.

If it's not an issue with the timestep, other possibilities include issues in the BCs and/or meteorology fields. Do these high concentrations appear at a particular day and time consistently? That could warrant looking into the input files for any corrupted data.

**Comment by lfreese** [comment]
@msulprizio thanks-- this wiki link is very helpful! I'm encountering this across all levels, and it starts specifically on 20151226. When I changed the timestep to 5/10 transport/chemistry, the results on 20151226 no longer have negative values, so this does seem to potentially fix the issue (I've run it through 20151231). I'm running it out for a few more days to check results into 2016. 

If it is the case that it is a timestep issue, it might be helpful to include some sort of warning in future GC updates for those running nested grids that this may be a source of error on certain dates and need adjustment. I do see that in the conversation response to the wiki there was discussion on the increased amount of time it takes to do a 5/10 timestep run, so it shouldn't necessarily be automatically set to this, but it might be useful to keep record of the dates where the CFL condition is breaking due to meteo conditions (I wonder if any of the dates that you and Bram ran into this problem for are also dates I'm running into it). 

I checked the BC files and those don't seem to have any negative or zero values, although they do have small values in the northeast grid boxes (see example on 20151207), and I've previously checked the meteo fields and am not getting any corrupted data there either; which just helps back up that if the timestep change works it is likely due to that. 

I'm wondering if in the work you and Bram did you had to maintain the 5/10 timesteps, or if you just had certain dates for which you dropped the timestep down? I'm trying to determine if it is best to run the entire year on the 5/10 timestep or transition back at some point to reduce computing time.
